### PR TITLE
Created base navbar and base colors

### PR DESCRIPTION
**Pull request by yagolomondo**
- Added two color variables

- Changed logo on navbar

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <%= stylesheet_link_tag 'application', media: 'all' %>
     <%#= stylesheet_pack_tag 'application', media: 'all' %> <!-- Uncomment if you import CSS in app/javascript/packs/application.js -->
   </head>
-  <body>
+  <body style="background-color: #F4FBFF; color: #000411;">
```
Comment (obrenoco) [inline app/views/layouts/application.html.erb]: EVITE FAZER INLINE STYLE NO BODY !
STYLES NÃO TEM EFEITO !

Comment (yagolomondo) [reply app/views/layouts/application.html.erb]: > :::: tl:dr: pode apagar o Style ::::
> EVITE FAZER INLINE STYLE NO BODY !
> STYLES NÃO TEM EFEITO !

O style estava funcionando normalmente quando eu testei.

File: app/assets/stylesheets/components/_navbar.scss [hunk 2]
```diff
@@ -10,3 +17,11 @@
 .navbar-lewagon .navbar-brand img {
   width: 40px;
 }
```
Comment (obrenoco) [inline app/assets/stylesheets/components/_navbar.scss]: TOGGLER (right buttom) not visible when page is shrinked.... 
solution:

ADD:    
.navbar-toggler {
     background-color: white;
}

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (obrenoco) [inline app/views/layouts/application.html.erb]: :::: tl:dr: pode apagar o Style ::::
EVITE FAZER INLINE STYLE NO BODY !
STYLES NÃO TEM EFEITO !

Comment (yagolomondo) [reply app/views/layouts/application.html.erb]: O style funciona quando eu testei

**Comment by elisanog** [comment]
I am happy you removed the sh**** light blue background color from the login page :) hehe

**Pull request merged**